lib/native-client.js

```diff
@@ -365,6 +365,25 @@ class NativeClient extends EventEmitter {
   }
 
   /**
+   * Execute an aggregation framework pipeline with the provided options on the
+   * collection.
+   *
```

Comment:
At this lower level, perhaps link to the node driver docs?
e.g. http://mongodb.github.io/node-mongodb-native/2.2/api/Collection.html#aggregate